controllers/validation/account_validation_controller.go

```diff
@@ -360,6 +361,10 @@ func (r *AccountValidationReconciler) Reconcile(ctx context.Context, request ctr
 		log.Error(err, "Could not retrieve account to validate", "account-request", request.NamespacedName)
```

Comment:
Can we also change this to a `log.Info` here?
```suggestion
		log.Info(err, "Account does not exist", "account-request", request.NamespacedName)
```